subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -228,6 +247,56 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	}
 #endif
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
+	struct lll_sync_iso_stream *stream;
+	struct ll_sync_iso_set *sync_iso;
+	isoal_sink_handle_t sink_handle;
+	struct lll_sync_iso *lll_iso;
+	struct ll_iso_datapath *dp;
+	uint32_t sdu_interval;
+	uint16_t iso_interval;
+	uint8_t burst_number;
+	int err;
+
+	if (path_dir != BT_HCI_DATAPATH_DIR_CTLR_TO_HOST) {
+		return 0;
```

Comment:
Shouldn't we return an error in this case?

Otherwise I also suggest that we use the HCI macros for sucess since the error messages are HCI:
```suggestion
		return BT_HCI_ERR_SUCCESS;
```

Reply:
Updated to return error.